### ISSUE TITLE
可否提供Regdb数据集上的log?

### ISSUE THREAD

**Issue by ZilinZou** [nengdong96/CSDN#8]
请问可以提供RegDB数据集上的log吗？我在SYSU数据集上复现出了对应的精度，但在RegDB数据集上始终复现不出来，v-t的R1只有82.23%。或许是不同数据集上有些参数做了修改吗？

我在RegDB数据集上的参数设置如下：
Namespace(auto_resume_training_from_lastest_step=True, batch_size=32, cuda='cuda', dataset='regdb', eval_epoch=1, gall_mode='single', img_h=288, img_w=144, lambda1=0.15, lambda2=0.05, lambda3=0.1, learning_rate=0.0003, loss=1, max_save_model_num=1, milestones=[40, 70], mode='train', num_pos=4, num_workers=8, output_path='output/regdb', pid_num=206, regdb_data_path='../../Datasets/RegDB/', regdb_test_mode='v-t', resume_test_model=119, resume_train_epoch=-1, seed=1, stage1_batch_size=32, stage1_learning_rate=0.0003, stage1_lr_min=1e-06, stage1_train_epochs=60, stage1_warmup_epochs=5, stage1_warmup_lr_init=1e-05, stage1_weight_decay=0.0001, stage2_learning_rate=0.0003, sysu_data_path='../../Datasets/SYSU-MM01/', test_mode='all', total_train_epoch=120, trial=1, weight_decay=0.0005)

best_epoch65: R1: 0.822330117225647, R10: 0.9626213312149048, R20: 0.9820388555526733, mAP: 0.7626481214772624

**Comment by fengjiayu888** [comment]
> 请问可以提供RegDB数据集上的log吗？我在SYSU数据集上复现出了对应的精度，但在RegDB数据集上始终复现不出来，v-t的R1只有82.23%。或许是不同数据集上有些参数做了修改吗？
> 
> 我在RegDB数据集上的参数设置如下： Namespace(auto_resume_training_from_lastest_step=True, batch_size=32, cuda='cuda', dataset='regdb', eval_epoch=1, gall_mode='single', img_h=288, img_w=144, lambda1=0.15, lambda2=0.05, lambda3=0.1, learning_rate=0.0003, loss=1, max_save_model_num=1, milestones=[40, 70], mode='train', num_pos=4, num_workers=8, output_path='output/regdb', pid_num=206, regdb_data_path='../../Datasets/RegDB/', regdb_test_mode='v-t', resume_test_model=119, resume_train_epoch=-1, seed=1, stage1_batch_size=32, stage1_learning_rate=0.0003, stage1_lr_min=1e-06, stage1_train_epochs=60, stage1_warmup_epochs=5, stage1_warmup_lr_init=1e-05, stage1_weight_decay=0.0001, stage2_learning_rate=0.0003, sysu_data_path='../../Datasets/SYSU-MM01/', test_mode='all', total_train_epoch=120, trial=1, weight_decay=0.0005)
> 
> best_epoch65: R1: 0.822330117225647, R10: 0.9626213312149048, R20: 0.9820388555526733, mAP: 0.7626481214772624

你好，请问在SYSU数据集上复现出对应精度，你是怎么设置的环境和参数呢？为什么我复现出来的结果很低

**Comment by oiooi111** [comment]
@ZilinZou 你好 我遇到了同样的问题，regdb数据集复现不出来，请问你现在复现出来了吗，可以请教请教吗

**Comment by jazzzjh** [comment]
> [@ZilinZou](https://github.com/ZilinZou) 你好 我遇到了同样的问题，regdb数据集复现不出来，请问你现在复现出来了吗，可以请教请教吗

我也复现不出来。
